spec/reek/cli/command/todo_list_command_spec.rb

```diff
@@ -37,6 +37,40 @@
         result = command.execute
         expect(result).to eq(Reek::CLI::Options::DEFAULT_SUCCESS_EXIT_CODE)
       end
+
+      it 'writes a todo file' do
+        command.execute
+        expected_yaml = { 'FeatureEnvy' => { 'exclude' => ['self'] } }.to_yaml
```

Comment:
Sure, I have nothing against using the hash - but the values in that hash shouldn't be literal but instead come from `attributes_for` ...